k8s/prod/cluster-00-overlay/em/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: em
+bases:
+- ../../../namespaces/em
+- ../../../namespaces/em/em-icp/prod.yaml
```

Comment:
You should add this to patches and actual manifest in bases

Reply:
but wouldn't that add it to all environments?

Reply:
It should be like : https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/cluster-00-overlay/rd/kustomization.yaml#L5-L8

Reply:
done